IPs/RSKIP180.md

```diff
@@ -0,0 +1,61 @@
+# Limit the RSK merged mining merkle proof
+
+
+|RSKIP          | 180 |
+| :------------ |:-------------|
+|**Title**      |Limit the RSK merged mining merkle proof|
+|**Created**    |15-JUL-2021 |
+|**Author**     |VK |
+|**Purpose**    |Sec |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+In this RSKIP we propose to limit the size of the bitcoin merged mining merkle proof field in a RSK block header.
+This check should happen during the block validation step.
+
+## Motivation
+
+The RSKj merge mining is compatible with the bitcoin protocol. This is: a miner can mine both bitcoin and RSK simultaneously.
+But they are not required to do so. They can mine RSK blocks with an invalid BTC header.
+
+In this case, the RSK merge mining merkle proof inclusion can be arbitrarily large. In particular, the merkle tree can have any height.
+And while this is the case, it is unreal for a BTC block to have a height higher than 30.
```

Comment:
Unclear .. you want to restrict height of Merkle tree to realistic levels.. but last sentence is restricting height of "BTC block"

Reply:
@Vovchyk can you see this comment please?

Reply:
This PR has stagnated for long time. Given the recent review and updates by @aeidelman, I think we can merge it *as is* so further discussions can happen on the research forum.